package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@kiali/kiali-ui",
   "version": "0.8.0",
+  "proxy": "https://kiali-istio-system.192.168.122.2.nip.io",
```

Comment:
You don't want this!

Reply:
lol, whoops, I can't believe I didn't catch that before commit.